docs/source/model_api.rst

```diff
@@ -2,6 +2,7 @@
 The Model API
 **************
 
-.. currentmodule:: cellmlmanip.model
 
-.. autoclass :: Model
+.. automodule:: cellmlmanip.model
+
+.. autofunction:: cellmlmanip.rdf.create_rdf_node
```

Comment:
Is this the only `rdf` function that needs documenting? If so maybe it doesn't need its own module?

Reply:
It is currently. But I think conceptually it's useful to keep it separate anyway? We don't have a utils module or similar.